supabase/migrations/20260410074000_add_triage_to_clinical_case_states.sql

```
@@ -0,0 +1,22 @@
+
+-- Index for searching urgent cases
+CREATE INDEX IF NOT EXISTS idx_clinical_case_states_triage_priority
+  ON public.clinical_case_states(triage_priority)
+  WHERE triage_priority != 'routine';
+
+-- Update existing records by extracting from state JSONB if possible
+UPDATE public.clinical_case_states
+SET 
+  triage_priority = COALESCE(state->'triage'->>'priority', 'routine'),
+  triage_mandate = state->'triage'->>'mandate'
+WHERE state ? 'triage';
```

SQLFluff (4.1.0)

[error] Line 8, Position 12: Found unparsable section: 'IF NOT EXISTS triage_priority TEXT NOT N...'

(PRS)